### PR TITLE
[react-big-calendar] Replace unused prop with correct prop and type

### PR DESCRIPTION
**Pull request by ldthorne**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/intljusticemission/react-big-calendar/blob/master/src/Calendar.js#L738 and https://intljusticemission.github.io/react-big-calendar/examples/index.html#prop-getNow

### CODE REVIEW

**Review by eps1lon — APPROVED** [review]

File: types/react-big-calendar/react-big-calendar-tests.tsx [hunk 1]
```diff
@@ -68,63 +68,65 @@ class CalendarResource {
...
-                  eventWrapper: EventWrapper,
-              }}
-              dayPropGetter={customDayPropGetter}
-              slotPropGetter={customSlotPropGetter}
-              defaultDate={new Date()}
-              resources={getResources()}
-              resourceAccessor={event => event.resourceId}
-              resourceIdAccessor={resource => resource.id}
-              resourceTitleAccessor={resource => resource.title} />
+                <MyCalendar  {...this.props}
```
Comment (eps1lon) [inline types/react-big-calendar/react-big-calendar-tests.tsx]: Could you revert the initial indentation? It creates an unnecessarily big diff that's hard to parse and hurts git blame functionality.

Comment (ldthorne) [reply types/react-big-calendar/react-big-calendar-tests.tsx]: @eps1lon Done. Thanks for the review!
